### PR TITLE
chore: Modify the current time to Utc time

### PR DESCRIPTION
**Pull request by zhenlei520**

### CODE REVIEW

File: src/DDD/MASA.BuildingBlocks.DDD.Domain/Entities/Auditing/AuditEntity.cs [hunk 1]
```diff
@@ -5,11 +5,11 @@ public abstract class AuditEntity<TUserId> : Entity, IAuditEntity<TUserId>
 
     public TUserId Creator { get; set; } = default!;
 
-    public DateTime CreationTime { get; set; } = DateTime.Now;
+    public DateTime CreationTime { get; set; } = DateTime.UtcNow;
```
Comment (doddgu) [inline src/DDD/MASA.BuildingBlocks.DDD.Domain/Entities/Auditing/AuditEntity.cs:8]: Utc seems reasonable, but I probably don't need it. What do I do?

File: src/Dispatcher/MASA.BuildingBlocks.Dispatcher.IntegrationEvents/Logs/IntegrationEventLog.cs [hunk 2]
```diff
@@ -17,7 +17,7 @@ public class IntegrationEventLog
 
     public int TimesSent { get; set; } = 0;
 
-    public DateTime CreationTime { get; private set; } = DateTime.Now;
+    public DateTime CreationTime { get; private set; } = DateTime.UtcNow;
 
     public string Content { get; private set; } = null!;
```
Comment (doddgu) [inline src/Dispatcher/MASA.BuildingBlocks.Dispatcher.IntegrationEvents/Logs/IntegrationEventLog.cs:23]: fix `Unchanged files with check annotations`

**Review by doddgu — COMMENTED** [review]
1

File: src/DDD/MASA.BuildingBlocks.DDD.Domain/Entities/Auditing/AuditEntity.cs (hunk 1 above)
Comment (zhenlei520) [reply src/DDD/MASA.BuildingBlocks.DDD.Domain/Entities/Auditing/AuditEntity.cs:8]: Reassign CreationTime or we provide a GetCurrentTime() for easy overloading

Comment (doddgu) [reply src/DDD/MASA.BuildingBlocks.DDD.Domain/Entities/Auditing/AuditEntity.cs:8]: The current PR resolve it or next?

File: test/MASA.BuildingBlocks.DDD.Domain.Tests/TestEntity.cs [hunk 3]
```diff
@@ -42,7 +42,7 @@ public void TestOperator()
         Assert.IsTrue(x == y);
         Assert.IsTrue(x != z);
 
-        MasaEntity m = null;
+        MasaEntity? m = null;
```
Comment (doddgu) [inline test/MASA.BuildingBlocks.DDD.Domain.Tests/TestEntity.cs:45]: Why utils called `MASA`, then called `Masa` here?

File: src/DDD/MASA.BuildingBlocks.DDD.Domain/Entities/Auditing/AuditEntity.cs [hunk 4]
```diff
@@ -18,9 +29,19 @@ public class AuditEntity<TKey, TUserId> : Entity<TKey>, IAuditEntity<TUserId>
...
```
Comment (doddgu) [inline src/DDD/MASA.BuildingBlocks.DDD.Domain/Entities/Auditing/AuditEntity.cs:32]: Remove the ` = DateTime.UtcNow;`

File: src/Dispatcher/MASA.BuildingBlocks.Dispatcher.IntegrationEvents/Logs/IntegrationEventLog.cs [hunk 5]
```diff
@@ -17,7 +17,7 @@ public class IntegrationEventLog
 
     public int TimesSent { get; set; } = 0;
 
-    public DateTime CreationTime { get; private set; } = DateTime.Now;
+    public DateTime CreationTime { get; private set; } = DateTime.UtcNow;
```
Comment (doddgu) [inline src/Dispatcher/MASA.BuildingBlocks.Dispatcher.IntegrationEvents/Logs/IntegrationEventLog.cs:20]: Remove the ` = DateTime.UtcNow;`, and add `GetCurrentTime`

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**